src/Application.js

```diff
@@ -1,15 +1,33 @@
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+
+import Paths from 'process/routes/paths';
+import { privateRoutes, publicRoutes } from 'process/routes';
 
-import routes from 'process/routes';
 import { Footer, Navbar } from 'show/components';
```

Comment:
This is grouped with the above. It's an absolute import

Reply:
done